### PR TITLE
Grid columns position

### PR DESCRIPTION
**Pull request by popovserhii**

### CODE REVIEW

File: src/ZfcDatagrid/Datagrid.php [hunk 1]
```diff
@@ -676,7 +681,28 @@ public function setColumns(array $columns)
     public function addColumn($col)
     {
         $col = $this->createColumn($col);
+
+        if (!$col->getPosition()) {
+            $col->setPosition(1000);
```
Comment (kokspflanze) [inline src/ZfcDatagrid/Datagrid.php]: why not as default `0`?

Comment (popovserhii) [reply src/ZfcDatagrid/Datagrid.php]: I use `ksort()` for ascending order. If set default as `0` we should then use negative values `-1, -10,...` for put the column at first position.

There is no convenient way in ZF which value use as default. I've read the different resource, such as [Ocramius comment on stackoverflow](https://stackoverflow.com/a/15122837/1335142) and [route documentation](https://framework.zend.com/manual/2.1/en/modules/zend.mvc.mvc-event.html#mvcevent-event-bootstrap) and do not found an answer which value is better.

Comment (kokspflanze) [reply src/ZfcDatagrid/Datagrid.php]: in the eventmanager the default priority is 1, see [zend-eventmanager](https://github.com/zendframework/zend-eventmanager/blob/master/src/EventManager.php#L185), so i would also recommend a lower number like 1.

about the negative values, you have to set a position, if you want it on an other place or work with negative values. looks for me like a normal workflow.

**Comment by kokspflanze** [comment]
@popovserhii thanks=)

**Pull request merged**